src/score.js

```diff
@@ -114,14 +114,11 @@ export const makeCandidateScoreCalc = (config, hostname, chatDomain, apiSlug, ap
 
         const userBadgeIds = items.map(mapToId);
 
-        const [badge] = items;
-
-        //TODO: why use badges for that if we pass an instance of User?
```

Comment:
Was this because user account on the election site may not be the same as the one linked in chat (Parented account)? Linked account rep may be different from the election site rep?

Reply:
judging from the logs, I think you are actually right, I thought we resolve reputation based on the account belonging to the election site (we should probably consider doing just that, or creating a <site, reputation> map since we fetch that information either way). I am going to revert that change and go sit in the corner for not realising it